suite.go

```diff
@@ -68,6 +68,42 @@ type suite struct {
 	afterScenarioHandlers  []AfterScenarioHook
 }
 
+type Attachment struct {
```

Comment:
Disagree on this one.

This aliasing isn't into user space and in any case except for GherkinDocument the alising is only the pickles.

Also using mesaages.Attachment would mean taking an internal message class and exposing the end user to it, which will may increate coupling. Also there is no reduction in data mapping as the cuke report has it's own data structures to populate and because there is no other usage of those message classes in godog as there is no support for the Messaages format yet (I have a separate PR coming for that).